### PR TITLE
respect indent style preference

### PR DESCRIPTION
**Pull request by noneofyourbusiness1415252**
Make yapf respect indent style preference
tested @ https://replit.com/@UMARismyname/black#f.py

### CODE REVIEW

**Comment by noneofyourbusiness1415252** [comment]
> ping me for a review once you do that

@jackyzha0 ✅

**Comment by jackyzha0** [comment]
could you uncommit the `package-lock.json`? seems like you got some package bumps in there that are not relevant to your changes

File: packages/pyright/webpack.config.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = (_, { mode }) => {
             timings: true,
         },
         resolve: {
-            extensions: ['.ts', '.js'],
+            extensions: ['.ts', '.js', '.node'],
```
Comment (jackyzha0) [inline packages/pyright/webpack.config.js:39]: these changes should also be removed :eyes: rest looks good though, I'll stamp for now and I'll merge once this is also cleaned up

**Review by jackyzha0 — APPROVED** [review]

**Comment by jackyzha0** [comment]
don't forget to run prettier! CI is failing on style check

**Pull request merged**